### PR TITLE
button ui component fixed

### PR DESCRIPTION
**Pull request by CrhistianT7**

### CODE REVIEW

File: src/ui/Button/Button.styles.ts [hunk 1]
```diff
@@ -63,7 +78,7 @@ export const StyledButton = styled.button<IStyledButton>`
   font-size: 16px;
```
Comment (GustavoUT22) [inline src/ui/Button/Button.styles.ts:78]: 1rem please

File: src/ui/Button/Button.styles.ts [hunk 2]
```diff
@@ -7,30 +7,45 @@ interface IStyledButton {
...
   switch (type) {
     case 'primary':
       return `
         color: ${colors.white.main};
-        background-color: ${colors.secondary.main};
+        background-color: ${colors.primary.main};
 
         &:focus {
-          outline: 2px solid ${colors.secondary.focus};
+          ouline: 2px solid ${colors.secondary.focus};
```
Comment (GustavoUT22) [inline src/ui/Button/Button.styles.ts:21]: change to outline

File: src/ui/Button/Button.styles.ts (hunk 1 above)
Comment (CrhistianT7) [reply src/ui/Button/Button.styles.ts]: done

File: src/ui/Button/Button.styles.ts (hunk 2 above)
Comment (CrhistianT7) [reply src/ui/Button/Button.styles.ts]: done

**Review by GustavoUT22 — APPROVED** [review]
Good Job!

**Pull request merged**